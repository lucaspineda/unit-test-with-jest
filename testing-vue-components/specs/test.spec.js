import TestComponent from '@/test'
import List from '@/list'
import { mount, shallowMount } from '@vue/test-utils'

test('mount', () => {
  const wrapper = mount(TestComponent, {
    propsData: {
      value: 'Another Test'
    }
  })
  expect(wrapper.html()).toMatchSnapshot()
});

test('ListComponent shallow', () => {
  console.log(mount(List).html())
  console.log(shallowMount(List).html())
});