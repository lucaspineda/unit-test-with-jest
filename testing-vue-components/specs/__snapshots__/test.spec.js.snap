// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`mount 1`] = `
<div>
  <h1>Hello Another Test</h1>
</div>
`;
